internal/models/payment_initiation_adjustments_status.go

```diff
@@ -168,3 +169,48 @@ func FromPaymentToPaymentInitiationAdjustment(from *Payment, piID PaymentInitiat
 		Error:     err,
 	}
 }
+
+func FromPaymentDataToPaymentInitiationAdjustment(status PaymentStatus, createdAt time.Time, piID PaymentInitiationID) *PaymentInitiationAdjustment {
```

Comment:
Why is this showing up as brand new in the diff? We had this logic before right?

Reply:
Let's delete the `FromPaymentToPaymentInitiationAdjustment` function above then so we don't have two nearly identical functions?